connector/pom.xml

```diff
@@ -159,7 +159,7 @@
         <dependency>
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-azure</artifactId>
-            <version>3.2.0</version>
+            <version>2.7.2</version>
```

Comment:
What happened here?

Reply:
couldnt make it work with 3.2.0 in the CI-CD 
This is provided anyway